hawk/app/assets/stylesheets/_theme.scss

```diff
@@ -11,7 +11,7 @@ $simulator-right: #80c235;
 
 // ----  content.scss variables - login page ----
 $darkbar-bg: #333;
-$darkbar-heading-fg:#00b268;
+$darkbar-heading-fg: rgb(251, 196, 66);
```

Comment:
<img width="816" alt="screen shot 2018-01-14 at 9 28 38 am" src="https://user-images.githubusercontent.com/320303/34916932-62a1026e-f90d-11e7-90da-927a6b4a9fff.png">

Fix the RGB to match the logo, please.